tests/models/test_dpt.py

```diff
@@ -0,0 +1,276 @@
+import os
+import pytest
+import inspect
+import tempfile
+from functools import lru_cache
+from huggingface_hub import hf_hub_download
+import torch
+import segmentation_models_pytorch as smp
+
+from tests.models import base
+from tests.utils import (
+    default_device,
+    requires_torch_greater_or_equal,
+    check_run_test_on_diff_or_main,
+)
+
+
+class TestDPTModel(base.BaseModelTester):
+    test_encoder_name = "tu-vit_tiny_patch16_224"
+    files_for_diff = [r"decoders/dpt/", r"base/"]
+
+    default_height = 224
+    default_width = 224
+
+    # should be overriden
+    test_model_type = "dpt"
+
+    @property
+    def hub_checkpoint(self):
+        return f"smp-test-models/{self.model_type}-tu-vit_tiny_patch16_224"
+
+    @property
+    def model_class(self):
+        return smp.MODEL_ARCHITECTURES_MAPPING[self.model_type]
+
+    @property
+    def decoder_channels(self):
+        signature = inspect.signature(self.model_class)
+        # check if decoder_channels is in the signature
+        if "decoder_channels" in signature.parameters:
+            return signature.parameters["decoder_channels"].default
+        return None
+
+    @lru_cache
+    def _get_sample(self, batch_size=None, num_channels=None, height=None, width=None):
+        batch_size = batch_size or self.default_batch_size
+        num_channels = num_channels or self.default_num_channels
+        height = height or self.default_height
+        width = width or self.default_width
+        return torch.rand(batch_size, num_channels, height, width)
+
+    @lru_cache
+    def get_default_model(self):
+        model = smp.create_model(
+            self.model_type, self.test_encoder_name, output_stride=None
+        )
```

Comment:
Can we avoid passing `output_stride=None` to make it consistent with other models? It can be handled on `create_model` level or encoder level for example